### ISSUE TITLE
Workflow is not triggered when changes are made to the PR

### ISSUE THREAD

**Issue by luisabez** [thehanimo/pr-title-checker#44]
I'm using this yaml file config, but the workflow is not automatically triggered when I edit the title. Once it's wrong, it can't be fixed and the workflow fails.

```
name: Lint PR
run-name: Lint PR

on:
  pull_request_target:
    types:
      - opened
      - edited
      - synchronize
      - labeled
      - unlabeled
  merge_group:

jobs:
  lint-pr:
    name: Lint PR

  check:
    runs-on: ubuntu-latest
    steps:

      - name: Check PR title
        uses: thehanimo/pr-title-checker@v1.4.1
        with:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
